core/jobs/src/main/java/com/cognifide/aet/job/common/collectors/screen/ScreenCollector.java

```diff
@@ -83,15 +97,62 @@ private boolean isPatternAndResultMD5Identical(byte[] screenshot) {
 
   @Override
   public void setParameters(Map<String, String> params) throws ParametersException {
-    //No parameters here
+    if (StringUtils.isNotBlank(params.get(XPATH_PARAM)) || StringUtils
```

Comment:
it is, but this is extra check for screen collector as both of those params are optional.